Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/InteractableInspector.cs

```diff
@@ -89,16 +89,17 @@ public virtual void RenderCustomInspector()
             bool showStates = false;
             SerializedProperty states = serializedObject.FindProperty("States");
             bool drawerStarted = false;
+            string statesPrefKey = "Settings_States";
```

Comment:
Was it necessary to bring out these two statements into this scope? It looks like these are still only used inside the if clause of line 95